examples/se3_pose_estimation/make_dataset_KingsCollege.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
This should probably live in the example directory. It's not really a minimal example